nilearn/datasets/func.py

```diff
@@ -2902,9 +2902,9 @@ def _download_data_spm_multimodal(data_dir, subject_dir, subject_id):
 
 def _make_events_filepath_spm_multimodal_fmri(_subject_data, session):
     key = f"trials_ses{session}"
-    events_file_location = os.path.dirname(_subject_data[key])
+    events_file_location = Path(_subject_data[key]).parent
     events_filename = f"session{session}_events.tsv"
-    events_filepath = os.path.join(events_file_location, events_filename)
+    events_filepath = events_file_location / events_filename
```

Comment:
```suggestion
    events_filepath = str(events_file_location / events_filename)
```

this would otherwise change to `Path` the type that is ultimately returned by

```python
from nilearn.datasets.func import fetch_spm_multimodal_fmri
data = fetch_spm_multimodal_fmri()
data.events1
```

I think that if we decide that our fetchers should return Path instead of strings, we should try to make the change in synchronized manner.